src/Notifications/NotificationsViewExtension.cs

```diff
@@ -98,6 +99,16 @@ public void Loaded(ViewLoadedParams viewStartupParams)
             (notificationsMenuItem.MenuItem.Parent as ContentControl).Content = null;
             //place the menu into the DynamoMenu
             viewStartupParams.dynamoMenu.Items.Add(notificationsMenuItem.MenuItem);
+
+            LoadNotificationCenter();
+        }
+
+        private void LoadNotificationCenter()
+        {
+            var dynamoView = (DynamoView)viewLoadedParams.DynamoWindow;
+            var shortcutBar = dynamoView.ShortcutBar;
+            var notificationsButton = (Button)shortcutBar.FindName("notificationsButton");
+            var notificationCenterController = new NotificationCenterController(dynamoView, notificationsButton);
```

Comment:
Maybe we can just add the button as a property to the new class? If add button members into the constructor, it might be inconvenient later for new changes